imgui-examples/examples/test_window_impl.rs

```diff
@@ -117,6 +118,7 @@ impl Default for State {
             stacked_modals_item: 0,
             stacked_modals_color: [0.4, 0.7, 0.0, 0.5],
             app_log: Vec::new(),
+            filter: TextFilter::new(String::from("Test")),
```

Comment:
I've checked other implementations of the bindings without ever seeing anything identical.
When I read the provided methods, I thought it would be better to have a pointer generated once for ImGuiTextFilter (here `raw`, generated by `imgui_sys::ImGuiTextFilter_ImGuiTextFilter`).

However, your idea is more in the thought of the crate but I don't know if the created pointer will be destroyed at each frame ?

Reply:
Marking as resolved - your current implementation is consistent with the C++ API, and there's no obvious reason to change it!